test/test_helper.exs

```diff
@@ -12,4 +12,22 @@ defmodule TestMacros do
       Decimal.new(unquote(str))
     end
   end
+
+  defmacro dbl_min(sign) do
+    quote do
+      %Decimal{sign: unquote(sign), coef: 22_250_738_585_072_014, exp: -324}
+    end
+  end
+
+  defmacro zero(sign) do
+    quote do
+      %Decimal{sign: unquote(sign), coef: 0, exp: 0}
+    end
+  end
+
+  defmacro dbl_max(sign) do
+    quote do
+      %Decimal{sign: unquote(sign), coef: 17_976_931_348_623_158, exp: 292}
+    end
+  end
```

Comment:
We don't use these macros afaict.

Reply:
Refactored test is now using dbl_min and dbl_max macros:
https://github.com/ericmj/decimal/pull/207/commits/760e56e88117a3ecc649c7fc46dbfd66ea6d1b91